### ISSUE TITLE
Darksiders Genesis (710920) - No higher than low textures for NVIDIA and AMD

### ISSUE THREAD

**Issue by Patola** [ValveSoftware/Proton#5951]
# Compatibility Report
- Name of the game with compatibility issues: Darksiders Genesis
- Steam AppID of the game: 710920

## System Information
- GPU: RX 6800 XT (System 1, a desktop PC) with 16 GB VRAM, RTX 3070 Mobile (System 2, a laptop PC) with 8 GB VRAM
- Driver/LLVM version: Mesa 22.1.2 (System 1), NVIDIA 515.48.07 (System 2)
- Kernel version: 5.18.8-tkg-bmq (System 1), 5.18.7-262-tkg-bmq (System 2)
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/Patola/e51e3af1be3c7d233b8f14fb21be1aa6 (System 1), https://gist.github.com/Patola/4f8dba624e97724218fb5f1650c292c6 (System 2)
- Proton version: 7.0-3 (also tried with GE-Proton7-16, same results)

## I confirm:
- [X] that I haven't found an existing compatibility report for this game. -- I have found an existing compatibility report from 2019 with no activity for a long time and with completely different issues.
- [X] that I have checked whether there are updates for my system available.

## Symptoms 
- Darksiders Genesis will hang up at the loading screen when starting a new game or loading an existing savegame if the "Textures" options is "mid" or higher (see screenshot).

## Reproduction
- Load the game.
- On graphics options, set the textures option to "mid" or higher (seen screenshot). The other graphics options do not make a difference. Also tried with all display modes (windowed, bordless windowed, fullscreen) and they don't make a difference. Different resolutions also do not make a difference. Tried on two systems.
- The game will hang when starting a new game or load an existing game.
- With the textures set to low, it will fully work.
- For the record, the game works on my Steam Deck in 1280x800 even with all graphics options set to Ultra, but since it has low VRAM it might just be that the game ignores that and loads low-res textures.

The attached logfiles are for the desktop PC (system 1). Two proton logs, one when loading a savegame with textures set to low (steam-710920-low-textures.log) and the other one with the textures set to mid, trying to load the same savegame.

For the note, running `htop` via ssh on the system at the time. When I try to load the savegame, processor usage goes to 100% on all processors and shortly the machine stops responding. This happens with the laptop and desktop PCs. `dmesg` does not show any message when this happens.

[steam-710920-mid-textures-hang.log](https://github.com/ValveSoftware/Proton/files/9029332/steam-710920-mid-textures-hang.log)
[steam-710920-low-textures.log](https://github.com/ValveSoftware/Proton/files/9029334/steam-710920-low-textures.log)
![screenshot](https://user-images.githubusercontent.com/2991620/176907656-ee89e00c-0645-4e92-8fa9-cf9436be9557.png)

**Comment by Patola** [comment]
Might be of note that once I realised this was happening, I disabled the Steam overlay to prevent any interference from other factors. I am also using icewm on both systems, a window manager that does not have any compositor.
Also I tried changing motion blur and v-sync and they do not make any difference.
The two hosts continue having a "ping" reply when they hang, but any new attempt to ssh to them hangs up.
